crates/bevy_core/src/time/time.rs

```diff
@@ -1,6 +1,18 @@
-use bevy_ecs::system::ResMut;
+use async_channel::{Receiver, Sender};
+use bevy_ecs::system::{Res, ResMut};
 use bevy_utils::{Duration, Instant};
 
+/// channel resource used to receive time from render world
+pub struct TimeReceiver(pub Receiver<Instant>);
+/// channel resource used to send time from render world
+pub struct TimeSender(pub Sender<Instant>);
+
+/// create channels used for sending time between render world and app world
+pub fn create_time_channels() -> (TimeSender, TimeReceiver) {
```

Comment:
Ditto here on why this is pub.

Reply:
this is pub because it's being using in the render workspace.